app/models/attraction.rb

```diff
@@ -0,0 +1,41 @@
+class Attraction
+  include Mongoid::Document
+  include Mongoid::Timestamps
+
+  field :name, type: String
+  field :rating, type: Integer
+  field :review_count, type: Integer
+  field :longlat, type: Hash
+  field :yelp_id, type: String
+  field :yelp_categories
+  field :yelp_mobile_url, type: String
+  field :yelp_url, type: String
+  field :yelp_image_url, type: String
+
+  belongs_to :location
+  has_and_belongs_to_many :categories
+
+  validates_presence_of :name, :longlat, :yelp_id
+  validates_uniqueness_of :yelp_id
+
+  def longitude
```

Comment:
:+1: to these nice methods!

But you shouldn't need to use `self` for reading.